NOTICE

```diff
@@ -0,0 +1,10 @@
+Parse Server
+
+Copyright 2015-present Parse Platform
+
+This product includes software developed at Parse Platform.
+www.parseplatform.org
+
+---
+
+As of April 5, 2017, Parse, LLC has transferred this code to the Parse Platform organization, and will no longer be contributing to or distributing this code.
```

Comment:
Parse Dart / Flutter SDK

Copyright 2021-present Parse Platform

This product includes software developed at Parse Platform.
www.parseplatform.org

---

As of October 19, 2021, the original author Phill Wiggins has transferred this code to the Parse Platform organization.